Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -135,10 +136,6 @@ sub _releases {
 	}
 
 	my @items;
-	my $searchTags = [
-		"artist_id:$artistId",
-		"library_id:$library_id",
-	];
```

Comment:
But that would remove the library filter, wouldn't it? We might _potentially_ end up with unwanted tracks which are part of the album, but _not_ of the current library.

Or the contributor filter: you'd get all composers of an album, not only the one we are coming from, right?

Reply:
Correct on both counts. On the first point, if it can happen, it will someday. On the second, I should have caught that in testing.

Changes pushed.